### ISSUE TITLE
No Mining Nodes in Roland's Doom (Duskwood)

### ISSUE THREAD

**Issue by JamesGent** [azerothcore/azerothcore-wotlk#4694]
No Mining Nodes inside Roland's Doom (Duskwood) 4 nodes outside.

![WoWScrnShot_030321_011756](https://user-images.githubusercontent.com/61194190/109867821-de1acb00-7c5e-11eb-8114-8a45d0acef17.jpg)

##### CURRENT BEHAVIOUR:

There are no mining nodes inside the mine called Roland's Doom in Duskwood.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

There should be at least a couple mining nodes inside Roland's Doom. You can see some here in this video.

https://www.youtube.com/watch?v=-I-m9yqiYCQ

Maybe we should look at TrinityCore's nodes for all herbs/mines? There are many funky places for nodes such as;

1) Durotar near Sen'jin village (40 nodes in a small area).
2) Teldrassil (very few herbs on the entire zone).
3) Roguefeather Den in Thousand Needles (40+ iron nodes in a single cave).

##### STEPS TO REPRODUCE THE PROBLEM:

1. Learn Find Minerals
2. Go to Roland's Doom
3. Look for mining nodes

**Comment by Zoidwaffle** [comment]
I haven't checked TC for these nodes, but for other zones they are exactly the same as AC. Which makes them both wrong (at least in some cases).

I'm currently rewriting all the nodes for Bloodmyst Isle. It had a whopping 8 herbs, 3 mineral spawns and 0 treasures for an entire zone.

It's super boring to rewrite this stuff, but perhaps I could look into Duskwood as one of the next areas.

**Comment by T1ti** [comment]
For vanilla content, remember to check vmangos, they worked on that a lot more than TC.

**Comment by Zoidwaffle** [comment]
I looked a bit into this.

The mineral spawns definitely got some love already - there are ~400 potential spawns and 112 pools... that are then combined into a master pool with 20 spawns.

I increased the amount of spawns - and found 2 mineral veins inside the mine.

So I don't think this is wrong as such - some tweaking might be fine, like:

```
UPDATE `pool_template` SET `max_limit`=30 WHERE `entry`=2009;
```

and to be fancy, make a "pool_pool" segment for the mine outside the rest of the zone, with perhaps 4 spawns (I didn't check how many spawn locations the mine have).
